README.md

```diff
@@ -26,15 +22,49 @@ Pass `wrednetdbg=0` to disable the NetDbg functionality.
+This project is under active research and development and is not yet functional but close to being so. Its goal is to enable AMD iGPU support for Renoir/Raven(2) and their derivatives, such as Picasso. However, due to the complexity of GPU drivers, especially those without public source code, extended support for non-existent logic is not possible. As a result, you must use Catalina (minimum) to Big Sur (recommended) since some required iGPU logic has been removed in Monterey.
+
+Injecting kernel extensions is not possible during the OpenCore injection stage.
+
+
+## **Introduction**
+
+We are a team of three working on getting graphics acceleration for AMD iGPUs (Raven/Raven2/Renoir and their derivatives) on hackintoshes (PCs running macOS). Currently, we are fixing the missing and broken logic by patching the existing kexts (AMDRadeonX5000 for GCN 5 and AMDRadeonX6000 for VCN).
+
+## **Progress**
+
+- Controller, accelerator and framebuffer are all attached and enabled.
+- GFX engine and SDMA0 engine start up and are able to process command buffers.
+- However, the data being fed to the Virtual Memory Page Table (VMPT) is likely incorrect, causing Page Faults in both GFX and ME as soon as WindowServer tries to clear the video memory.
+- We are investigating the cause of the Page Faults and will eventually fix it.
+
+## **ETA**
+
+Unknown, the project will be completed when it is done.
+
+## **Installation**
+
+It may be possible to install macOS on your laptop, but without graphics acceleration and the touchpad may not work.
+
+## **EFI Setup**
+
+We do not provide support for EFI setup, please follow the AMD section of the Dortania guide and use MacBookPro15,4 as SMBIOS.
+
+## **X5000 vs X6000**
+
+- X5000: GFX 9, uses VCE + UVD, same VBIOS logic as our device
+- X6000: GFX 10, uses VCN 2, newer VBIOS structure (displays may not be detected properly)
 
-Its aim is to enable AMD iGPU support for Renoir/Raven(2) and their derivatives, like Picasso.
+## **Current Status (X5000)**
 
-Unfortunately, due to the complexity of GPU drivers, especially ones that do not contain public source code distributions, extended support for non-existent logic is not possible. Therefore, you are required to use Catalina (Minimum) to Big Sur (recommended) as it appears the small part of the required iGPU logic has been removed since Monterey.
+- Everything starts out well and the AMD driver even acknowledges the progress.
+- However, as soon as WindowServer tries to zero the video memory, Page Faults occur and cause the SDMA0 engine to freeze, breaking the GFX engine and the entire acceleration process.
+- The AMD driver will attempt to reset the GFX/SDMA0 channels every minute, but the Page Faults still occur even after the reset.
+- This results in a black screen for a few minutes, before the machine eventually restarts due to the driver failing to properly reset the channels.
```

Comment:
```suggestion
- This results in a black screen for a few minutes, before the machine eventually restarts due to a watchdog timeout because WindowServer fails to check in success.
```